src/main/java/de/tum/cit/aet/artemis/lecture/domain/AttachmentUnit.java

```diff
@@ -32,6 +32,10 @@ public class AttachmentUnit extends LectureUnit {
     @JsonIgnoreProperties("attachmentUnit")
     private List<Slide> slides = new ArrayList<>();
 
+    @OneToOne(mappedBy = "correspondingAttachmentUnit")
```

Comment:
is this bidirectional linking necessary/beneficial or would a unidirectional link be more suitable?

Reply:
Also should this be lazy?